src/runtime/store/index.ts

```diff
@@ -201,4 +201,6 @@ export function derived<T>(stores: Stores, fn: Function, initial_value?: T): Rea
  * Get the current value from a store by subscribing and immediately unsubscribing.
  * @param store readable
  */
-export { get_store_value as get };
\ No newline at end of file
+export const get = get_store_value as {
```

Comment:
Would typing the `get_store_value` in `svelte/internal` be better?

Reply:
Yes definitely, but i did this for minimal changes.

Anyway, i think importing `Readable` interface inside `utils.ts` requires a little restructure, Cause there is a dependencies from `svelte/internal` which makes `npm run tsd` to fail.

I was thinking about to move all `runtime/*` interfaces to `runtime/interfaces.ts` just like `compiler`, hmm?

Reply:
Probably, yes. I've just merged #5269 and am going to close this. Thank you, though!